back/db/migrate/20200527094026_add_ordering_to_projects_topics.rb

```diff
@@ -1,10 +1,19 @@
 class AddOrderingToProjectsTopics < ActiveRecord::Migration[6.0]
-  def change
+  class StubProjectsTopic < ActiveRecord::Base
+    self.table_name = 'projects_topics'
+  end
+
+  def up
     add_timestamps :projects_topics
     add_column :projects_topics, :ordering, :integer
 
-    ProjectsTopic.order(:created_at).each.with_index do |projects_topic, index|
+    StubProjectsTopic.order(:created_at).each.with_index do |projects_topic, index|
```

Comment:
Good catch with `down` method. Indeed, rollback wouldn't work in this migration. To make it a bit simpler and more reliable, you can also leave `change` method (not splitting into up/down) and use reversible migration https://edgeguides.rubyonrails.org/active_record_migrations.html#using-reversible

Reply:
Done. Thanks, I didn't know about this method.